packages/riipen-ui/src/components/Tooltip.jsx

```diff
@@ -99,24 +105,39 @@ class Tooltip extends React.Component {
     const theme = this.context;
 
     return css.resolve`
-      .container {
-        padding: ${theme.spacing(4)}px;
-      }
-
       .wrapper {
         height: max-content;
         width: max-content;
       }
 
       .popover {
-        opacity: 0;
-        transition: opacity 0.3s ease-out;
-        visibility: hidden;
+        border-radius: 2px;
```

Comment:
Match design radius and update deafult colour to be white, like the web tooltips.